includes/amandaBackupDB.service-template

```diff
@@ -1,6 +1,7 @@
 [Unit]
-Description=nightly backup: Amanda config, DB, logs
+Description=Make nightly backup of Amanda database
 After=network.target network-online.target
+Wants=amandaBackupDB.timer
 
```

Comment:
tested ? useful ? Haven't seen those before

Reply:
Yes just a safer and more specific way of referring to timers